src/lib/parse/forbiddenChars.cpp

```diff
@@ -60,7 +60,7 @@ bool forbiddenChars(const char* s, const char* exceptions)
     case ';':
     case '(':
     case ')':
-      LM_E(("Bad Input (character '%c')", *s));
+      // LM_W(("Bad Input (character '%c')", *s));
```

Comment:
Yeah, I left this one here (as LM_W, not LM_E) as it is still in the air whether "level 1" or "level 2" LM_Ws should removed to avoid "double logs"